go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/spf13/cobra v0.0.6
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.4.0
-	github.com/vmware-tanzu/astrolabe v0.2.0
+	github.com/vmware-tanzu/astrolabe v0.1.2-0.20210226062805-d433214638d2
```

Comment:
I wonder why this is v0.1.2-xxxx, not v0.2.0-xxxx.  The latest release on astrolabe is v0.2.0.

Reply:
I have no idea. It is automatically generated by running `go get github.com/vmware-tanzu/astrolabe@d433214638d2f14038a1db08f3ec51bf7cf3f1a3`. However, the commit `d433214638d2` actually matches though.

Reply:
OK